.ci/updatecli/values.d/updatecli-compose.yaml

```diff
@@ -0,0 +1,3 @@
+spec:
```

Comment:
This might cause some issues, see https://github.com/elastic/apm-agent-go/blob/7a1371799ed890f8db4c6473d98afaf442d5005e/update-compose.yaml#L22

Reply:
I think we don't need to rename the values.d files but the content only to point to the `updatecli-compose.yaml` in the root folder

Reply:
Ah, missed that. thank you.

Good I only ran the script for this repo.

Reply:
oh okay.

Reply:
No worries, i also missed to review this condition in the PR 👼

Reply:
I'm gonna close this and re-run the script.